concourse/tasks/smoke-test.yml

```diff
@@ -2,18 +2,23 @@ platform: linux
 image_resource:
   type: registry-image
   source:
-    repository: governmentpaas/curl-ssl
+    repository: gdscyber/concourse-chrome-driver
+    tag: latest
 params:
-  URL:
-  MESSAGE:
+  CGO_ENABLE: "0"
+  DEBIAN_FRONTEND: "noninteractive"
+  PYTHONIOENCODING: "UTF-8"
+inputs:
+- name: git-master
+outputs:
+- name: builds
 run:
-  path: sh
+  path: /bin/bash
   args:
-    - '-c'
+    - -euo
+    - pipefail
+    - -c
     - |
-      set -eu
-
-      echo "$MESSAGE"
-
-      # TODO we can come up with a more thorough test than this
-      curl --fail --silent --verbose "$URL"
+      echo "running smoke tests against \"${WEB_APP_BASE_URL}\""
+      behave behave/features/ --tags='@e2e_happy_path_no_nhs_login' --stop
```

Comment:
there is a make command in the make file for this already ?

Reply:
Yeah, I had deliberately duplicated it here.

The Makefile target is called `concourse_e2e`—I've assumed that at some point the set of e2e tests will grow to cover more scenarios. I think for our smoke test it makes sense to just test one(?) happy path (so that we know that the app and DB are working).

I could maybe move this to a Makefile target called `smoke_test` or something?

Reply:
sounds good if you could just move it to a new target like you suggested